tests/conftest.py

```
@@ -9,128 +9,53 @@
 from unittest.mock import Mock, MagicMock, patch
 import tempfile
 import os
+import sys
+import stat
 import yaml
+import shutil
 from typing import Dict, List, Any, Optional
```

Flake8 (7.2.0)

[error] 'typing.Optional' imported but unused

(F401)